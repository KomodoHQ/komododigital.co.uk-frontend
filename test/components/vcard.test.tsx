import React from 'react';
import Vcard from '../../src/components/vcard';
import renderer from 'react-test-renderer';

test('Vcard element is rendered', () => {
  const component = renderer.create(
    <Vcard person="Armin" avatar={{
      childImageSharp: {
        fixed: {
          height: 29,
          src: "",
          srcSet: "",
          srcSetWebp: "",
          srcWebp: "",
          tracedSVG: "",
          width: 170
        }
      }
    }} />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
