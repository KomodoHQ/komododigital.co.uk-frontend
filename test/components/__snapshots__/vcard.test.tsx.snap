// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Vcard element is rendered 1`] = `
<div
  className="vcard"
>
  <div
    className=" gatsby-image-wrapper"
    style={
      Object {
        "display": "inline-block",
        "height": 29,
        "overflow": "hidden",
        "position": "relative",
        "width": 170,
      }
    }
  >
    
    <picture>
      
      <source
        srcSet=""
      />
      <img
        alt=""
        height={29}
        onLoad={[Function]}
        src=""
        style={
          Object {
            "height": "100%",
            "left": 0,
            "objectFit": "cover",
            "objectPosition": "center",
            "opacity": 1,
            "position": "absolute",
            "top": 0,
            "transition": "opacity 0.5s",
            "width": "100%",
          }
        }
        width={170}
      />
    </picture>
    <noscript
      dangerouslySetInnerHTML={
        Object {
          "__html": "<picture><img width=\\"170\\" height=\\"29\\" src=\\"\\" alt=\\"\\" style=\\"position:absolute;top:0;left:0;transition:opacity 0.5s;transition-delay:0.5s;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
        }
      }
    />
  </div>
  <div>
    <h1>
      Armin Talic
    </h1>
    <h2>
      Commercial Director
    </h2>
  </div>
</div>
`;
